frontend_v2/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I think `frontend-tutorial` is a better name to `frontend_v2`

Reply:
I am not sure that I'd call this a tutorial.